Makefile

```diff
@@ -68,4 +68,4 @@ install-katana:
 	cargo install --git https://github.com/dojoengine/dojo --locked --tag "${KATANA_VERSION}" katana
 
 run-katana:
-	katana --chain-id test --validate-max-steps 4000000 --invoke-max-steps 6000000 --eth-gas-price 0 --strk-gas-price 0 --disable-fee
+	katana --chain-id test --validate-max-steps 160000000 --invoke-max-steps 160000000 --eth-gas-price 0 --strk-gas-price 0 --disable-fee
```

Comment:
but we merged recently the opposite to be iso prod?

Reply:
Modifying this to be the requested prod setting: 
6M val. / 14M exec